.pre-commit-config.yaml

```diff
@@ -29,3 +29,9 @@ repos:
     hooks:
       - id: prettier
         args: [--prose-wrap=always, --print-width=88]
+
+  - repo: https://github.com/codespell-project/codespell
```

Comment:
I'm not sure adding this to pre-commit is worth it; it may just slow things down. Open to other opinions though.

Reply:
```
% time codespell --ignore-words-list="ans,hart,ot,poit,reallllly,tring" .

0.24s user 0.02s system 91% cpu 0.284 total
```
Could also run it in a GitHub Action.

Reply:
Dropped the pre-commit.